Base.rte/Scenes.ini

```diff
@@ -99,8 +99,8 @@ AddTerrainDebris = TerrainDebris
 AddTerrainDebris = TerrainDebris
 	PresetName = Small Cacti
 	DebrisFile = ContentFile
-		Path = Base.rte/Scenes/Objects/Cacti/CactusSmall.bmp
-	DebrisPieceCount = 16
+		Path = Base.rte/Scenes/Objects/Plants/CactusSmall.bmp
+	DebrisPieceCount = 15
```

Comment:
Why'd you change the DebrisPieceCount?

Reply:
From what I can tell it's because people missed there's still a SmallCactus015 (e.g. debris number 16) present, which should have been renamed to CactusSmall015.

Reply:
What Gotcha said.
Thanks Gotcha, i'll rename it and put the count back to 16.
That's the missing 16th bmp.